ark/spLDA/processing.py

```diff
@@ -0,0 +1,86 @@
+import functools
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import os
+import pickle
+from ark.utils.spatial_lda_utils import check_format_cell_table_args
+# from spatial_lda.featurization import *
+# from spatial_lda.visualization import plot_adjacency_graph()
+
+
+def format_cell_table(cell_table, markers=None, clusters=None, fovs="all_fovs"):
+    """
+    Formats a cell table containing one for more fields of view to be compatible with the spatial_lda library.
+    Args:
+        cell_table (pd.DataFrame):
+            A pandas DataFrame containing the columns of cell marker frequencies and/or cluster ids.
+        markers (list, str):
+            A list of strings corresponding to marker names.
+            Default: None
+        clusters (list, int):
+            A list of integers corresponding to cluster ids.
+            Default: None
+        fovs (str, int):
```

Comment:
Do you think the subsetting should happen here, or later in the pipeline? My gut would be later makes more sense, but I could easily be persuaded

Reply:
Either way works for me.  I figured if people knew they only wanted to run some FOVs then it just cuts down on the amount of runtime in the other steps.

Reply:
That's true. However, I think people will likely try training with different settings. So it'll be easiest if we preprocess everything first, since they'll eventually need this table anyway to run the model

Reply:
Sounds good